apps/tabs-storybook/src/app/wizard/wizard.component.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
I don't think we need this spec file that was auto-generated by our schematic. At some point we should look at not generating it. @johnhwhite